crates/rustc_codegen_spirv/src/builder/spirv_asm.rs

```diff
@@ -436,8 +492,8 @@ impl<'cx, 'tcx> Builder<'cx, 'tcx> {
         }
         if tokens.next().is_some() {
             self.tcx.sess.err(&format!(
-                "too many operands to instruction: {}",
-                instruction.class.opname
+                "too many operands to instruction: {} {:?}",
+                instruction.class.opname, instruction
```

Comment:
The debug impl includes the opname, probably best to print it only once (but also the debug impl is pretty gross, do we really want to include it in an error message?)

Reply:
TIL `opname`, is that like `format!("{:?}", instruction.class.opcode)`?

Reply:
This is debug left-over that I can remove.